drivers/clock_control/clock_control_nrf2_audiopll.c

```diff
@@ -0,0 +1,309 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT nordic_nrfs_audiopll
+
+#include "clock_control_nrf2_common.h"
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/clock_control/nrf_clock_control.h>
+#include <nrfs_audiopll.h>
+#include <nrfs_backend_ipc_service.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_DECLARE(clock_control_nrf2, CONFIG_CLOCK_CONTROL_LOG_LEVEL);
+
+#define SHIM_MIN_FREQUENCY 10666667
+#define SHIM_MAX_FREQUENCY 13333293
+
+#define SHIM_DEFAULT_PRESCALER AUDIOPLL_DIV_4
```

Comment:
This should be DIV_12 to get 10-13MHz range (4 is for AUXPLL range which is not configurable through the service and should not be changed for audiopll)